connector_ebisumart/models/ebisumart_sale_order_importer.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import Component
+from odoo.addons.connector.components.mapper import mapping
+from ..components.mapper import normalize_datetime
+
+
+class SaleOrderImportMapper(Component):
+    _name = 'ebisumart.sale.order.import.mapper'
+    _inherit = 'ebisumart.import.mapper'
+    _apply_on = ['ebisumart.sale.order']
+
+    direct = [
+        ("ORDER_NO", "external_id"),
+        ("ORDER_DISP_NO", "name"),
+        (normalize_datetime('REGIST_DATE'), 'created_at'),
+        (normalize_datetime('UPDATE_DATE'), 'updated_at'),
+    ]
+    children = [('order_details', 'ebisumart_order_line_ids', 'ebisumart.sale.order.line')]
+
+    @mapping
+    def partner_id(self, record):
+        for line in record.get('order_details', []):
+            binder = self.binder_for('ebisumart.product.product')
+            product = binder.to_internal(line['ITEM_ID'], unwrap=True)
+            if product and product.torihikisaki_id != 0:
+                partner = self.env["res.partner"].search([('ebisumart_id','=', product.torihikisaki_id),('customer','=',True)],limit=1)
+                return {'partner_id': partner.id}
+
+    @mapping
+    def journal_id(self, record):
+        if record['KESSAI_ID'] == 61:
+            journal = self.env["account.journal"].search([("ebisumart_payment_type", "=", "credit_card")],limit=1)
+            if journal:
+                return {'journal_id': journal.id}
+        else:
+            journal = self.env["account.journal"].search([("ebisumart_payment_type", "=", "payment_slip")],limit=1)
+            if journal:
+                return {'journal_id': journal.id}
+        return
+    
+    # TODO
+    # To search process id with some identifier, not name
+    @mapping
+    def workflow_process_id(self, record):
+        workflow_process_id = self.env["sale.workflow.process"].search([("name", "=", "Automatic")],limit=1)
```

Comment:
How about adding the data for this part in this module itself?

It means to add like:
https://github.com/OCA/sale-workflow/blob/2a951aa90b52ac74dbbe1898aceea4d0af72ff9d/sale_automatic_workflow/data/automatic_workflow_data.xml#L44-L59

It should be changed to <field name="validate_picking" eval="1" />. But otherwise use the same parameters as "Automatic".